docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -30,7 +30,7 @@ helm list
 
 ```bash
 NAME	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART           	APP VERSION
-demo	default  	1       	2021-02-18 23:34:14.547364974 +0000 UTC	deployed	k8ssandra-0.51.0	3.11.10     
+demo	default  	1       	2021-03-04 15:21:41.573192403 +0000 UTC	deployed	k8ssandra-0.55.0	3.11.10     
```

Comment:
Could we update this to reflect the 1.0.0 chart?

Reply:
Yeah will do.  My env is a little old.

Reply:
Updated